packaging/torcharrow/meta.yaml

```diff
@@ -20,20 +20,22 @@ requirements:
     - typing
     - tabulate
     - pyarrow
-    - arrow
+    - arrow-cpp==8.0.0
     - cffi
-    - glog==0.4.0
+    - glog==0.6.0
```

Comment:
anything changed on Velox side?

Reply:
I think velox always uses the glog installed from brew, which was recently updated from 0.5.0 to 0.6.0. Building with glog0.6.0 and running with glog0.4.0 will not work. However, the conda default channel hasn't updated to 0.6.0 yet, it only exists on conda-forge, hence this change.

Reply:
This is also why we need to specify `-c conda -forge` when installing.